connectivity/model/src/main/java/org/eclipse/ditto/connectivity/model/OAuthClientCredentials.java

```diff
@@ -115,6 +128,8 @@ public JsonObject toJson() {
         jsonObjectBuilder.set(JsonFields.CLIENT_ID, clientId, Objects::nonNull);
         jsonObjectBuilder.set(JsonFields.CLIENT_SECRET, clientSecret, Objects::nonNull);
         jsonObjectBuilder.set(JsonFields.REQUESTED_SCOPES, requestedScopes, Objects::nonNull);
+        jsonObjectBuilder.set(JsonFields.REQUESTED_SCOPES, requestedScopes, Objects::nonNull);
```

Comment:
this line is duplicated and needs to be removed

Reply:
removed it